### ISSUE TITLE
[Feature] Ability to remove incorrect fingerprints from scenes

### ISSUE THREAD

**Issue by kronkels** [stashapp/stash-box#304]
**Is your feature request related to a problem?**
Stash's tagger may match scenes incorrectly due to incorrect fingerprints associated with scenes on stash-box.

**Describe the solution you'd like**
Make it possible to remove incorrect hashes directly in stash-box. In addition, when a new scene is added with hashes that already belong to another scene, one could ask the user to automatically file a removal of the incorrect hashes.

**Describe alternatives you've considered**
Keep using the backlog spreadsheet, which is inefficient.

**Comment by peolic** [comment]
Just to add to this,
in case anyone is interested in removing incorrect fingerprints they submitted __themselves__, #115 added that option to the API, but it's not visible on any UI currently.
note that you can only unmatch fingerprints submitted using your user __after__ January 24th, 2022 (when stashdb was updated)

the GQL mutation:
```gql
mutation {
  submitFingerprint(input: {
    unmatch: true
    scene_id: "SCENE_ID"
    fingerprint: {
      hash: "FINGERPRINT_HASH"
      algorithm: PHASH  # or OSHASH, or MD5
      duration: 500  # duration in seconds (hover over fingerprint duration on stashdb)
    }
  })
}
```

**Comment by JackDawson94** [comment]
Like I said on Discord a couple days ago.
We will run into the issue of people re-submitting the bad fingerprint because the scene is still wrongly associated in their Stash.

So this should include a notion of blocking deleted hashes from being re-submitted I guess ?

**Comment by VonTittyslappen** [comment]
> Like I said on Discord a couple days ago. We will run into the issue of people re-submitting the bad fingerprint because the scene is still wrongly associated in their Stash.
> 
> So this should include a notion of blocking deleted hashes from being re-submitted I guess ?

Yep, agree with you, some flag functionality is required so if people do try to submit the wrong fingerprint against the scene, something at StashDB end would check if the flag is active and if so report an error message to the person submitting the fingerprint.

**Comment by neru2132** [comment]
In my opinion it should be possible to remove fingerprints through the regular scene edit dialog.
These edits should be considered sensitive operations just like other DELETE or MERGE operations.
In the case of StashDB or other large community driven stash-boxes, voters should be briefed to only vote YES on such edits if the reason for deleting the fingerprints is properly documented and sound.
Blocking previously removed fingerprints from re-submission outright is bound to create problems sooner or later, but a warning in stash similar to the one you see when the duration is off might be useful.

**Comment by Maista6969** [comment]
#1006 added the ability for moderators to move and delete fingerprints so unless we'd like to change that to an edit operation that any user can submit then this can probably be closed

**Comment by Shadesbird** [comment]
I just created https://github.com/stashapp/stash/issues/7058, which might be an alternative way to solve incorrect fingerprints without the need for an admin / moderator to manually ban them.

*Edit: Issue is now in stash instead of stash-box
